interface-definitions/containers.xml.in

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="container" owner="${vyos_conf_scripts_dir}/containers.py">
+    <properties>
+      <help>Container applications</help>
+    </properties>
+    <children>
+      <tagNode name="name">
+        <properties>
+          <help>Container name</help>
+        </properties>
+        <children>
+          <leafNode name="allow-host-networks">
+            <properties>
+              <help>Allow host networks in container</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="description">
+            <properties>
+              <help>Container description</help>
+            </properties>
+          </leafNode>
+          <leafNode name="image">
+            <properties>
+              <help>Image name in the hub-registry</help>
+            </properties>
+          </leafNode>
+          <tagNode name="network">
+            <properties>
+              <help>Attach user defined network to container</help>
+              <completionHelp>
+                <path>container network</path>
+              </completionHelp>
+            </properties>
+            <children>
+              <leafNode name="address">
+                <properties>
```

Comment:
Is there no IPv6 addressing possible? If this is possible, please use this building block: https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/interface/address-ipv4-ipv6.xml.i

Reply:
It possible for podman itself, 

but I don't have this function for CLI. Because I use only "ipv4-prefix" only.

I tried DUAL ipv4 + ipv6 for one network name, but I get a different ipv4 prefix. 
For example, I set **192.0.2.0** for ipv4
```
podman network create NET --subnet=192.0.2.0/24 --subnet=2001:db8::/64 --ipv6
```
And get random ipv4 range
```
                    "ranges": [
                        [
                            {
                                "gateway": "2001:db8::1",
                                "subnet": "2001:db8::/64"
                            }
                        ],
                        [
                            {
                                "gateway": "10.88.2.1",
                                "subnet": "10.88.2.0/24"
                            }
                        ]
                    ],
```

Ipv4 only shows correct network
```
root@r4-roll:/home/vyos# podman network create NET99 --subnet=192.168.99.0/24
/etc/cni/net.d/NET99.conflist

```
Check range
```
                    "ranges": [
                        [
                            {
                                "gateway": "192.168.99.1",
                                "subnet": "192.168.99.0/24"
                            }
                        ]
                    ],
```

Reply:
Maybe a podman issue? Works perfectly on Docker

```bash
$ docker network create NET --subnet=192.0.2.0/24 --subnet=2001:db8::/64 --ipv6
2dfe2f7fb25e3e6a9530b1d00b3ce6a8ca1e3de091812cdfafa901ba9feb5ace
$ docker network inspect NET
[
    {
        "Name": "NET",
        "Id": "2dfe2f7fb25e3e6a9530b1d00b3ce6a8ca1e3de091812cdfafa901ba9feb5ace",
        "Created": "2021-04-12T21:31:48.364178638+02:00",
        "Scope": "local",
        "Driver": "bridge",
        "EnableIPv6": true,
        "IPAM": {
            "Driver": "default",
            "Options": {},
            "Config": [
                {
                    "Subnet": "192.0.2.0/24"
                },
                {
                    "Subnet": "2001:db8::/64"
                }
            ]
        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": {},
        "Options": {},
        "Labels": {}
    }
]
```